src/ensembl/production/metadata/grpc/service.py

```diff
@@ -24,9 +35,15 @@ def serve():
     )
     server.add_insecure_port("[::]:50051")
```

Comment:
Wait.... I would as well revise this, why insecure ? (we use ssl in our cluster - so should be configurable)

Reply:
It has been like this since I picked up the code

When trying to use `add_secure_port()` I get the following error (which is expected)
```
File "src/ensembl/production/metadata/grpc/service.py", line 37, in serve
    server.add_secure_port("[::]:50051")
TypeError: add_secure_port() missing 1 required positional argument: 'server_credentials'
```
We need to get a certificate for it to work, and the new code will look something like this (I'm keeping it here for reference)
```
def serve():
    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
    ensembl_metadata_pb2_grpc.add_EnsemblMetadataServicer_to_server(
        EnsemblMetadataServicer(), server
    )

    # Load server-side SSL credentials
    with open('server.crt', 'rb') as f:
        server_certificate = f.read()
    with open('server.key', 'rb') as f:
        server_private_key = f.read()

    # Create server SSL credentials
    server_credentials = grpc.ssl_server_credentials(
        ((server_private_key, server_certificate,),)
    )

    # Add secure port using SSL credentials
    server.add_secure_port('[::]:50051', server_credentials)
    ...
```

Reply:
Ok. For Q2 then!